### DISCUSSION TITLE
Trying to install...

### DISCUSSION THREAD

**Discussion by stigges** [cnr-isti-vclab/TagLab#73] [General]
Hi there, 

I am probably (definitely) doing something wrong, but I am getting this when I try to open TagLab:

Traceback (most recent call last):
  File "C:\Users\stigges\Documents\TagLab-main\taglab.py", line 43, in <module>
    from source.QtAlignmentToolWidget import QtAlignmentToolWidget
ModuleNotFoundError: No module named 'source'

I have checked my dependencies and I think I have everything squared away, but I could be missing something. Does anyone know what the issue is?

Thanks!

-Sophia

**Comment by maxcorsini** [comment]
Hi Stigges,

to try to understand what is happening tell us your OS, python/CUDA version, and the installation steps that you have followed.

Best,
Massimiliano

**Comment by stigges** [comment]
Hi Massimiliano, 

I am using Windows 10, python version 3.9.13, CUDA version11.6. 

I have followed the instructions at https://github.com/cnr-isti-vclab/TagLab/wiki/Install-TagLab as well as making sure to install the dependencies in accordance with the "dependencies" note in the Taglab-main folder.

Thanks, 
Sophia

**Comment by stigges** [comment]
Hi again, 

I have still not been able to resolve this issue. It looks like all of the dependencies are installed, but I am still getting the same error: 

Traceback (most recent call last):
  File "C:\Users\stigges\Documents\TagLab-main\taglab.py", line 43, in <module>
    from source.QtAlignmentToolWidget import QtAlignmentToolWidget
ModuleNotFoundError: No module named 'source'

Any idea what the issue could be?

Thanks!

Best, 
Sophia
